addons/website_event_meet/static/src/js/snippets/options.js

```diff
@@ -31,7 +36,15 @@ options.registry.WebsiteEvent.include({
     async _computeWidgetState(methodName, params) {
         switch (methodName) {
             case 'allowRoomCreation': {
-                return this._getRpcData('meeting_room_allow_creation');
+                const rpcData = await this._rpc({
+                    model: "event.event",
+                    method: 'read',
+                    args: [
+                        [this.eventId],
+                        ['meeting_room_allow_creation'],
+                    ],
+                });
+                return rpcData[0]['meeting_room_allow_creation'];
```

Comment:
It seems like the state is being inverted somehow. i.e. it displays the "room creation" elements when the toggle is off.

I think this is because of my suggestion to do the query here, I didn't realize we would reload and this would be updated.
I think you can revert to doing the RPC in the start and getting it here sorry :/

@PierreCourt